addons/admissions-webhook/admissions-webhook.addon

```diff
@@ -0,0 +1,22 @@
+# Name: admissions-webhook
```

Comment:
Maybe a different name is needed for this.

Reply:
I think this name is fine, as `minishift addon enable admissions-webhook` looks good.